lib/vkloader.c

```diff
@@ -287,7 +287,9 @@ optimalTilingCallback(int miplevel, int face,
     user_cbdata_optimal* ud = (user_cbdata_optimal*)userdata;
 
     // Set up copy to destination region in final image
+#if defined(_DEBUG)
     assert(ud->region < ud->regionsArrayEnd);
+#endif
```

Comment:
How come only these asserts caused problems?

Reply:
Because `ud->regionsArrayEnd` only exist in debug builds.
In general I only followed up on build errors, not on asserts per se.